### DISCUSSION TITLE
What App-Choreo needs your help on...

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#26] [Team Posts]
###### Join our org [Here ✉️](https://github.com/App-Choreography/Get-An-Invite/issues/new?assignees=CodingSpecies&labels=Organisation+Invite%21+%F0%9F%93%A8&template=please-can-i-join-this-organisation------.md&title=Please+Can+I+Join+This+Organisation%3F+%F0%9F%A5%BA%F0%9F%99%8F") | [Our Website 🌐](https://app-choreography.github.io/) | [Repositories 📚](https://github.com/orgs/App-Choreography/repositories)

-------------------------------------------

## To @App-Choreography/community ,
We want to make this org, and it's repos, 🎉 **successful** 🎉 , so here is a list of things we will try to do (if you can help, please either comment on this thread, or open an issue in the support repo):

- [x] ~Add beta issue templates. [scroll to the creating issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)~
- [ ] Add automatic issue and pull request reply, using github actions.
- [ ] Get more branding 📛. [👉See repo👈](https://github.com/App-Choreography/Branding)
- [x] Add more to the website 🌐. [👉See repo👈](https://github.com/App-Choreography/app-choreography.github.io)

### Don't forget, you will be credited for every help you give!

Thank you.

**Comment by 0xMukesh** [comment]
@CodingSpecies Hey, I could work on the `issue templates` 

I will add it by 6 PM IST, I don't have access to my laptop right now 😅

**Comment by AvidCoder101** [comment]
@CodingSpecies I could try improving the website with a few additions for you if you would like? I will make a PR ASAP for it when I can. Thanks (:

**Comment by NS007-dev** [comment]
Thank you @AvidCoder101 and @kira272921
